checkQC/handlers/q30_handler.py

```diff
@@ -30,17 +30,24 @@ def check_qc(self):
 
         index_count = 0
         non_index_count = 0
+        lane_no = 1
```

Comment:
```suggestion
        lane_no = 0
```
If lane is set to one here, I think line 31-32 could be removed. Then the counters will be set in the for loop during the first iteration. If I'm not missing something. Let me know if you agree.

Reply:
Yes, I agree, that is brilliant.